contrib/packs/actions/pack_mgmt/unload.py

```diff
@@ -16,13 +15,7 @@ def __init__(self, config=None):
         self.initialize()
 
     def initialize(self):
```

Comment:
This was basically broken all along - we didn't correctly inherit parent arguments which means those actions never had access to the right config values (they always just access to the default ones).

Pack management API, here we come.